src/main/webapp/frame.jsp

```diff
@@ -51,7 +51,7 @@
           <div class="region-sidebar-left col-sm-2 col-xs-0 hidden-xs no-margin" ng-if="$storage.showSidebar && APP_FLAGS.showSidebar">
               <side-bar-menu></side-bar-menu>
           </div>
-          <div ng-if="(!($storage.showSidebar)) && APP_FLAGS.showSidebar" class="show-sidebar" ng-click="$storage.showSidebar = true">
+          <div ng-if="(!($storage.showSidebar)) && APP_FLAGS.showSidebar" class="show-sidebar" ng-click="$storage.showSidebar = true" onClick="pushGaEvent('Sidebar','Maximize Sidebar', 'Maximize Sidebar')">
```

Comment:
@timlevett Is there any distinct advantage to using the angular pushGaEvent method in the `miscService` over just using something like `onClick="_gaq.push(['_trackEvent', 'Beta', 'Read About Change', 'Read About Change']);"`? I can't remember if we talked about this. Seems like this would be much easier than pulling in the dependency anywhere we need to track events.